Sources/MapboxMaps/Annotations/ClusterOptions.swift

```diff
@@ -33,12 +33,30 @@ public struct ClusterOptions: Equatable {
     /// clustered points. Has the form `{"property_name": [operator, map_expression]}`.
     /// `operator` is any expression function that accepts at
     /// least 2 operands (e.g. `"+"` or `"max"`) — it accumulates the property value from clusters/points the
-    /// cluster contains; `map_expression` produces the value of a single point.
+    /// cluster contains; `map_expression` produces the value of a single point. Example:
     ///
-    /// Example: `{"sum": ["+", ["get", "scalerank"]]}`.
+    /// ``Expression`` syntax:
+    /// let expression = Exp(.sum) {
+    ///     Exp(.get) { "scalerank" }
+    /// }
+    /// clusterProperties: ["sum": expression]
```

Comment:
```suggestion
    /// ```
    /// let expression = Exp(.sum) {
    ///     Exp(.get) { "scalerank" }
    /// }
    /// clusterProperties: ["sum": expression]
    /// ```
```
you can fence code block by either 3-backsticks (`) or tildes (~)

Reply:
Updated in most recent commit